ui/selectors/selectors.js

```diff
@@ -546,13 +546,18 @@ export function getShowWhatsNewPopup(state) {
  * @returns {Object}
  */
 function getAllowedNotificationIds(state) {
+  const currentKeyring = getCurrentKeyring(state);
+  const currentKeyringIsLedger =
+    currentKeyring && currentKeyring.type === KEYRING_TYPES.LEDGER;
```

Comment:
Can this be `currentKeyring?.type === KEYRING_TYPES.LEDGER`

Reply:
done in 5919c1f0d